src/package.xml

```diff
@@ -1,21 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>Insert_Student_Details</members>
-        <members>StudentQuery</members>
-        <name>ApexClass</name>
-    </types>
     <types>
         <members>Student__c</members>
```

Comment:
Student Object is already deployed. remove it from the Package.xml

Reply:
student object modified For assignment 9

Reply:
If Student Object is modified for Assignment 9 then please add the changes for the student object in this PR.